src/applications/personalization/profile/tests/e2e/direct-deposit/gating.cypress.spec.js

```diff
@@ -74,33 +74,40 @@ function confirmDirectDepositIsBlocked() {
 }
 
 describe('Direct Deposit', () => {
+  let getPaymentInfoStub;
   beforeEach(() => {
     disableFTUXModals();
+    getPaymentInfoStub = cy.stub();
     cy.login();
     cy.route('GET', '/v0/feature_toggles*', dd4eduEnabled);
   });
   it('should be blocked if the user is not in EVSS and they are not signed up for DD4EDU', () => {
     cy.route('GET', 'v0/user', mockUserNotInEVSS);
     cy.route('GET', 'v0/profile/ch33_bank_accounts', mockDD4EDUNotEnrolled);
+    cy.intercept('v0/ppiu/payment_information', () => {
+      getPaymentInfoStub();
+    });
     cy.visit(PROFILE_PATHS.PROFILE_ROOT);
 
-    // TODO: add test to make sure that GET payment_information is not called?
```

Comment:
I remember making this comment and it drove me crazy to not know how to make a test for this! Feels good to figure it out.